src/context/providers/PointsGridFormDataProvider.js

```diff
@@ -39,6 +38,11 @@ export class PointsGridFormDataProvider extends mix(JSONSchemaFormDataProvider).
         return Math.floor(5 / this._divisor);
     }
 
+    get reciprocalVectorRatios() {
+        const lattice = new Made.ReciprocalLattice(this.material.lattice);
+        return lattice.reciprocalVectorRatios.map((r) => lodash.round(r, 3));
```

Comment:
We've been using `toPrecision` from code.js elsewhere, I think, maybe that would work better for consistency

Reply:
`toPrecision` returns a string, but the schema fields expect a number (the values are also needed for the calculation later). Converting the string back to a number here is obviously not a huge problem, but also not desirable. `Number.toPrecision` also does not fix the rounding problem for edge cases like `1.005` (there's a [SO post discussing](https://stackoverflow.com/questions/11832914/how-to-round-to-at-most-2-decimal-places-if-necessary) different options). I think we should either stick with rounding the number here, or use strings in the schema and convert them to float in the assignment unit.

Reply:
Let's use `code.js:numberToPrecision` as it was put there for this exact purpose